From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Lumine1909 <133463833+Lumine1909@users.noreply.github.com>
Date: Sat, 17 May 2025 17:36:22 +0800
Subject: [PATCH] Leaves: Replay Mod API

Original license: GPL-3.0-only
Original project: https://github.com/LeavesMC/Leaves

This patch is Powered by ReplayMod (https://github.com/ReplayMod)

diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
index a358376291a30301b2a8fe2f7fbf7e7c52f29ccf..40c61e6e9c2d225214f5636dc3170c0e4655b5e9 100644
--- a/src/main/java/org/bukkit/Bukkit.java
+++ b/src/main/java/org/bukkit/Bukkit.java
@@ -3219,4 +3219,10 @@ public final class Bukkit {
         server.clearBlockHighlights();
     }
     // Purpur end - Debug Marker API
+
+    // Leaves start - Photographer API
+    public static @NotNull org.leavesmc.leaves.entity.photographer.PhotographerManager getPhotographerManager() {
+        return server.getPhotographerManager();
+    }
+    // Leaves end - Photographer API
 }
diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index a7e6a92c3e329f5e831ea8526b9f8f6ef1908b13..8fdb9bc562c6e656cedd6602d2cce8e3a63eba57 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -2992,4 +2992,6 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
      */
     long getLastTickOversleepTime();
     // Gale end - YAPFA - last tick time - API
+
+    @NotNull org.leavesmc.leaves.entity.photographer.PhotographerManager getPhotographerManager(); // Leaves - Photographer API
 }
